Dockerfile

```diff
@@ -13,49 +15,50 @@ ENV TAUTULLI_DOCKER=True
 RUN \
   echo "**** install build packages ****" && \
   apk add --no-cache --virtual=build-dependencies \
-    g++ \
-    gcc \
-    make \
+    build-base \
+    cargo \
     python3-dev && \
   echo "**** install packages ****" && \
   apk add --no-cache \
-    curl \
-    jq \
-    py3-openssl \
-    py3-pip \
-    py3-setuptools \
+    git \
```

Comment:
I don't think git is needed as a runtime dep

Reply:
The build fails without it. I believe one of the python dependencies in tautulli's `requirements.txt` uses it. Also, it appears tautulli itself uses git to check for updates (granted we would not want updates inside the container).